runners/command-runner/src/main/scala/stryker4s/run/Stryker4sArgumentHandler.scala

```diff
@@ -0,0 +1,26 @@
+package stryker4s.run
+import grizzled.slf4j.Logging
+import org.apache.logging.log4j.Level
+import org.apache.logging.log4j.core.config.Configurator
+
+object Stryker4sArgumentHandler extends Logging {
+  private lazy val logLevels: Map[String, Level] =
+    Level.values().map(level => (level.toString.toLowerCase, level)).toMap
+
+  def parseArgs(args: Array[String]): Unit = {
```

Comment:
Can we find a better name for this? it's not only parsing args its also setting the loglevel

Reply:
I changed it to `handleArgs` for now. (which should make sense because the class is called a `Handler` too 😅 )